src/app/study/page.tsx

```
@@ -0,0 +1,86 @@
+import { CategoryCard } from "@/components/citizenship/category-card";
+import { Card, CardContent, CardDescription, CardHeader, CardTitle } from "@/components/ui/card";
+import { getAllCategories, getQuestionsByCategory, getRandomQuestions } from "@/lib/sample-questions";
```

ESLint

[error] 'getRandomQuestions' is defined but never used.

(@typescript-eslint/no-unused-vars)